Kuadrant/authorino-operator
# Authorino Reconciler

This PR is the starting point to enable the deployment of [Authorino](https://github.com/Kuadrant/authorino) instances via the operator.

Ps.: missing unit testing

## Verification steps

1) Clone the repo from this branch
2) Install the operator on a cluster of your choice
3) Run the operator eg: `make install run`
4) Create an instance of authorino using the CR, an example can be [found here](https://github.com/Kuadrant/authorino-operator/blob/foundation2/config/samples/authorino-operator_v1beta1_authorino.yaml).
5) Check if the authorino instance was deployed correctly by verifying whether a deployment is created and a pod is up and running
